docs/integrations/debugging.md

```diff
@@ -38,50 +39,75 @@ demisto.debug("I shouldn't have gone into STEM")
 This will print to the logs at the "Debug" level.
 
 And lastly:
+
 ```python
 demisto.error("I could open an Italian restaurant with all this spaghetti I am writing.")
 ```
+
 Will print to the logs at the "Error" level. It also may or may not notify your co-workers of your short comings.
 
-## Using Demistomock
-Sometimes when printing or using the logs is too confusing or messy you want to just use the debugger and go through the code line-by-line or breakpoint-by-breakpoint. This is where demistomock comes in. Ever noticed that all integrations in your IDE start with 
-```python 
+## Debugging using your IDE
+
+Sometimes when printing or using the logs is too confusing or messy you want to just use the debugger and go through the code line-by-line or breakpoint-by-breakpoint.
+
+> It is recommended to use the [Cortex XSOAR Visual Studio Code Extension](../concepts/vscode-extension.md) when you are developing content.
+
+### Python Environment  
+
+Python Environment  
```

Comment:
lets remove that section, it will confuse users with https://xsoar.pan.dev/docs/concepts/dev-setup